cluster/charts/rook-ceph/values.yaml

```diff
@@ -53,252 +53,268 @@ pspEnable: true
 
 ## Settings for whether to disable the drivers or other daemons if they are not
 ## needed
+csiEnable: true
 csi:
   enableRbdDriver: true
   enableCephfsDriver: true
   enableGrpcMetrics: true
   enableSnapshotter: true
   # (Optional) set user created priorityclassName for csi plugin pods.
-  # pluginPriorityClassName: system-node-critical
+  pluginPriorityClassName: system-node-critical
```

Comment:
We don't know if these priority classes even exist so I don't think we can set a default, especially on k8s 1.14 or older before priority classes were available.